wlutil/config.py

```diff
@@ -488,9 +472,9 @@ def __init__(self, cfgFile=None, cfgDict=None):
                 self.cfg['firesim']['simulation_inputs'] = [pathlib.Path(f) for f in self.cfg['firesim']['simulation_inputs']]
 
         if 'out-dir' in self.cfg:
-            self.cfg['out-dir'] = wlutil.getOpt('image-dir') / self.cfg['out-dir']
+            self.cfg['out-dir'] = wlutil.getOpt('image-dir') / wlutil.getOpt('board-dir').name / self.cfg['out-dir']
```

Comment:
Did you add an extra folder layer to the images? This `board-dir` to further distinguish images.

Reply:
Yes I'll add a note on this to the PR